testsuite/python/array_properties.py

```diff
@@ -268,12 +266,17 @@ def test_rot_aniso(self):
         self.set_copy(self.system.part[0].gamma_rot)
 
     def test_lb(self):
+        lbf = lb.LBFluid(agrid=0.5, dens=1, visc=1, tau=0.01)
+        self.system.actors.add(lbf)
+
         # Check for exception for various operators
         # LB
-        self.locked_operators(self.lbf[0, 0, 0].velocity)
-        self.locked_operators(self.lbf[0, 0, 0].stress)
-        self.locked_operators(self.lbf[0, 0, 0].stress_neq)
-        self.locked_operators(self.lbf[0, 0, 0].population)
+        self.locked_operators(lbf[0, 0, 0].velocity)
```

Comment:
why is it prohibited to set a velocity?

Reply:
and why does this test pass at all? There should only be an error raised if the shape is wrong

https://github.com/espressomd/espresso/blob/b89263db5db6e4084118fc4c54ba0f9068bcb9e5/src/python/espressomd/lb.pyx#L387-L400

Reply:
Sorry I do not understand the question. What do you mean by "prohibited to set a velocity"? Where does such an error occur?

Reply:
I think I partly understand your question now. But I am not the author of these lines of code, I just adapted them so they would run with the new setup. I cannot answer any questions on why they are there.

Reply:
I think I misunderstood the test... Probably the test should have some doc-strings what is being tested but you are right that this is not your business.